opensds-provisioner/pkg/client/client.go

```diff
@@ -83,8 +86,8 @@ func (c *SdsClient) Delete(volumeId string) error {
 	return c.client.DeleteVolume(volumeId, &model.VolumeSpec{})
 }
 
-// GetClient return OpenSDS Client
-func getSdsClient(endpoint string) *client.Client {
+// getSdsClient return OpenSDS Client
```

Comment:
I think `getsdsClient` method should be removed for competing the logic in `client/helper.go`.